### PR TITLE
Removes ASIMOV as a rollable and buildable lawset

### PR DESCRIPTION
**Pull request by Zergspower**
## About The Pull Request

After too many issues surrounding this lawset  we've decided to disable it from being buildable and rollable on the server. Originally it was a full removal but looking at the maintenance side of things it's better to just disable it with a few lines

## How This Contributes To The Skyrat Roleplay Experience

We've too many non-human and human-'like' races that stack onto the 'custom species' perk which made this a policy and administrative nightmare and simply didnt really add anything but annoyance to the players and borgs alike.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image
image

</details>

## Changelog

:cl:
add: Added in the missing default lawset board into the research node and made it printable
code:  Removed the ability to research/print ASIMOV and removed the weight to 0 for rolling.
/:cl:

### CODE REVIEW

File: config/game_options.txt [hunk 1]
```diff
@@ -239,7 +239,7 @@ DEFAULT_LAWS 0
 ## See datums\ai_laws.dm for the full law lists
 ## IE, SPECIFIED_LAWS asimovpp, SPECIFIED_LAWS robocop, SPECIFIED_LAWS antimov
 
-SPECIFIED_LAWS asimovpp
```
Comment (Zonespace27) [inline config/game_options.txt]: Revert everything that touches the config

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: config/game_options.txt (hunk 1 above)
Comment (Zergspower) [reply config/game_options.txt]: it was already in an edit but ok

File: config/game_options.txt [hunk 2]
```diff
@@ -249,7 +249,7 @@ SPECIFIED_LAWS asimovpp
 ## standard-ish laws. These are fairly ok to run
 #SKYRAT EDIT ADDITION BEGIN
 RANDOM_LAWS armadyne_safeguard
-RANDOM_LAWS asimov
+RANDOM_LAWS asmimov
```
Comment (Zonespace27) [inline config/game_options.txt:252]:
```suggestion
RANDOM_LAWS asimov
```

File: config/game_options.txt (hunk 1 above)
Comment (Zonespace27) [reply config/game_options.txt]: already done =/= good

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]
No this can be done through config.

**Comment by Zonespace27** [comment]
The lawset existing isn't a problem, if it being a roundstart (with the right station trait) is a problem, talk to a keyholder

**Comment by Zergspower** [comment]
> The lawset existing isn't a problem, if it being a roundstart (with the right station trait) is a problem, talk to a keyholder

This PR did more than just remove the boards, it made the current default buildable and researchable, on top of correcting the default text for the peacekeeper borg to remove mentions of human

**Pull request closed without merging**